gateway/helm/values.yaml

```diff
@@ -5,7 +5,7 @@ replicaCount: 1
 image:
   # Kubernetes image pull policy. Options: Always, IfNotPresent, Never.
   # https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
Why?

Reply:
it's better if the image tag is `latest`. Without `Always`, it will use whatever container is already pulled, which may not be the "latest".